sdk/translation/ai-translation-text/review/cognitiveservices-translator.api.md

```diff
@@ -0,0 +1,716 @@
+import { ClientOptions } from '@azure-rest/core-client';
+import { HttpResponse } from '@azure-rest/core-client';
+import { RawHttpHeaders } from '@azure/core-rest-pipeline';
+import { RawHttpHeadersInput } from '@azure/core-rest-pipeline';
+import { RequestParameters } from '@azure-rest/core-client';
+import { StreamableMethod } from '@azure-rest/core-client';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export interface AlignmentOutput {
+    proj: string;
+}
+
+// @public
+export interface BackTranslationOutput {
+    displayText: string;
+    frequencyCount: number;
+    normalizedText: string;
+    numExamples: number;
+}
+
+// @public (undocumented)
+export interface BreakSentence {
+    post(options: BreakSentenceParameters): StreamableMethod<BreakSentence200Response | BreakSentenceDefaultResponse>;
+}
+
+// @public (undocumented)
+export interface BreakSentence200Headers {
+    "x-requestid": string;
+}
+
+// @public
+export interface BreakSentence200Response extends HttpResponse {
+    // (undocumented)
+    body: Array<BreakSentenceElementOutput>;
+    // (undocumented)
+    headers: RawHttpHeaders & BreakSentence200Headers;
+    // (undocumented)
+    status: "200";
+}
+
+// @public (undocumented)
+export interface BreakSentenceBodyParam {
+    body: Array<InputTextElement>;
+}
+
+// @public (undocumented)
+export interface BreakSentenceDefaultHeaders {
+    "x-requestid": string;
+}
+
+// @public (undocumented)
+export interface BreakSentenceDefaultResponse extends HttpResponse {
+    // (undocumented)
+    body: MtErrorResponseOutput;
+    // (undocumented)
+    headers: RawHttpHeaders & BreakSentenceDefaultHeaders;
+    // (undocumented)
+    status: string;
+}
+
+// @public
+export interface BreakSentenceElementOutput {
+    detectedLanguage?: DetectedLanguageOutput;
+    sentLen: SentenceLengthOutput;
+}
+
+// @public (undocumented)
+export interface BreakSentenceHeaderParam {
+    // (undocumented)
+    headers?: RawHttpHeadersInput & BreakSentenceHeaders;
+}
+
+// @public (undocumented)
+export interface BreakSentenceHeaders {
+    "X-ClientTraceId"?: string;
+}
+
+// @public (undocumented)
+export type BreakSentenceParameters = BreakSentenceQueryParam & BreakSentenceHeaderParam & BreakSentenceBodyParam & RequestParameters;
+
+// @public (undocumented)
+export interface BreakSentenceQueryParam {
+    // (undocumented)
+    queryParameters?: BreakSentenceQueryParamProperties;
+}
+
+// @public (undocumented)
+export interface BreakSentenceQueryParamProperties {
+    language?: string;
+    script?: string;
+}
+
+// @public (undocumented)
+export function buildMultiCollection(queryParameters: string[], parameterName: string): string;
+
+// @public
+export interface CommonScriptModelOutput {
+    code: string;
+    dir: string;
+    name: string;
+    nativeName: string;
+}
+
+// Warning: (ae-forgotten-export) The symbol "TranslatorCustomEndpoint" needs to be exported by the entry point index.d.ts
```

Comment:
Could we export everything that needs to be public?

Reply:
What would be the command to do it?

Reply:
This has been addressed already and the warning is gone.